common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -251,30 +259,103 @@ def setUp(self, *args, **kwargs):
 
     @contextmanager
     def bulk_assertions(self):
+        """
+        A context manager that will capture all assertion failures made by self.assert*
+        methods within its context, and raise a single combined assertion error at
+        the end of the context.
+        """
         if self._manager:
             yield
```

Comment:
I changed this line to "1/0", and ran the tests, and all 50 passed.  What is this case for?